runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -4066,6 +4072,18 @@ J9::Z::CodeGenerator::inlineDirectCall(
          break;
       }
 
+   switch (methodSymbol->getRecognizedMethod())
```

Comment:
Should we check if we have vector support on the platform to generate inlined version of CRC32C? 
On platform where we do not have vector support enabled or available?